### ISSUE TITLE
Idea: Support Pinch-style Zoom and Other Trackpad Gestures

### ISSUE THREAD

**Issue by blackketter** [dexterleng/vimac#384]
Some apps, like Safari and Maps, respond to trackpad pinch-to-zoom gestures differently from two-fingered scroll. It would be great if scroll mode had keys that could emulate this.

Beyond this, other touchpad gestures could also be emulated but they are less common I think (swipes, rotation, 3+ fingers, smart zoom).

**Comment by dexterleng** [comment]
This is something I wish I could do!  Apps that support `Command +/-` is chunky and a bit disorientating.

One should be able to perform these gestures on arbitrary locations, although supporting them in center could be a start.

**Comment by blackketter** [comment]
Right! The center of the currently selected scroll area (when in scroll mode).  Or the current cursor position when in hint/mouse mode.

Is this even possible?  Or are the trackpad gestures private APIs?
